app/src/AddWordsComponents.js

```diff
@@ -14,7 +14,7 @@ class AddWordsComponent extends React.Component {
 		console.log(values);
 		const db = firebase.firestore();
 		const room = "someRoom"; // Replace this with getting dynamic room ID
-		var onSubmit = this.props.onSubmit;
+		const onSubmit = this.props.onSubmit; // why var?
```

Comment:
+1, everything possible should be marked as `const` (which is most things, in my experience)

Reply:
kk, will remove comment from this line